crates/stc_ts_file_analyzer/src/analyzer/types/mod.rs

```diff
@@ -2436,6 +2467,41 @@ impl Analyzer<'_, '_> {
                 }
             }
 
+            Type::Keyword(KeywordType {
+                kind: TsKeywordTypeKind::TsUnknownKeyword,
+                span,
+                ..
+            }) => {
+                if !self.rule().strict_null_checks {
+                    return;
+                }
+
+                let mut unknown = vec![
+                    Type::Keyword(KeywordType {
+                        span: *span,
+                        kind: TsKeywordTypeKind::TsNullKeyword,
+                        metadata: Default::default(),
+                        tracker: Default::default(),
+                    }),
+                    Type::Keyword(KeywordType {
+                        span: *span,
+                        kind: TsKeywordTypeKind::TsUndefinedKeyword,
+                        metadata: Default::default(),
+                        tracker: Default::default(),
+                    }),
+                    Type::TypeLit(TypeLit {
+                        span: *span,
+                        members: vec![],
+                        metadata: Default::default(),
+                        tracker: Default::default(),
+                    }),
+                ];
+                unknown.retain(|ty| !ty.type_eq(excluded.normalize()));
```

Comment:
You don't need to call `Type.normalize()` for `type_eq`

Reply:
thx, i fixed it